packages/ai-bot/package.json

```diff
@@ -2,14 +2,20 @@
     "name": "@cardstack/ai-bot",
     "devDependencies": {
         "matrix-js-sdk": "^25.0.0",
+        "openai": "4.0.0-beta.4",
         "ts-node": "^10.9.1",
         "typescript": "^5.1.6",
         "openai": "4.0.0-beta.4"
```

Comment:
hmm interesting how there is 2 openai dev dependencies here. Perhaps this was manually inserted

Reply:
Probably my mistake while merging updates from main, will fix.